IdentityCore/src/requests/broker/MSIDSSOExtensionSilentTokenRequest.m

```diff
@@ -25,47 +25,27 @@
 #import <AuthenticationServices/AuthenticationServices.h>
 #import "ASAuthorizationSingleSignOnProvider+MSIDExtensions.h"
 #import "MSIDSSOExtensionSilentTokenRequest.h"
-#import "MSIDSilentTokenRequest.h"
 #import "MSIDRequestParameters.h"
-#import "MSIDAccountIdentifier.h"
-#import "MSIDAuthority.h"
-#import "MSIDJsonSerializer.h"
 #import "ASAuthorizationSingleSignOnProvider+MSIDExtensions.h"
 #import "MSIDSSOExtensionTokenRequestDelegate.h"
 #import "MSIDBrokerOperationSilentTokenRequest.h"
 #import "NSDictionary+MSIDQueryItems.h"
-#import "MSIDOauth2Factory.h"
-#import "MSIDBrokerOperationTokenResponse.h"
-#import "MSIDIntuneEnrollmentIdsCache.h"
-#import "MSIDIntuneMAMResourcesCache.h"
-#import "MSIDSSOTokenResponseHandler.h"
-#import "MSIDThrottlingService.h"
-#import "MSIDDefaultTokenCacheAccessor.h"
 #import "ASAuthorizationController+MSIDExtensions.h"
 
-#if !EXCLUDE_FROM_MSALCPP
-#import "MSIDLastRequestTelemetry.h"
-#endif
-
 @interface MSIDSSOExtensionSilentTokenRequest () <ASAuthorizationControllerDelegate>
 
-@property (nonatomic) ASAuthorizationController *authorizationController;
 @property (nonatomic, copy) MSIDRequestCompletionBlock requestCompletionBlock;
-@property (nonatomic) id<MSIDCacheAccessor> tokenCache;
-@property (nonatomic) MSIDAccountMetadataCacheAccessor *accountMetadataCache;
+@property (nonatomic) MSIDBrokerOperationSilentTokenRequest *operationRequest;
+@property (nonatomic) ASAuthorizationController *authorizationController;
 @property (nonatomic) MSIDSSOExtensionTokenRequestDelegate *extensionDelegate;
 @property (nonatomic) ASAuthorizationSingleSignOnProvider *ssoProvider;
-@property (nonatomic, readonly) MSIDProviderType providerType;
-@property (nonatomic, readonly) MSIDIntuneEnrollmentIdsCache *enrollmentIdsCache;
-@property (nonatomic, readonly) MSIDIntuneMAMResourcesCache *mamResourcesCache;
-@property (nonatomic, readonly) MSIDSSOTokenResponseHandler *ssoTokenResponseHandler;
-@property (nonatomic) MSIDBrokerOperationSilentTokenRequest *operationRequest;
-@property (nonatomic) NSDate *requestSentDate;
 
 @end
 
 @implementation MSIDSSOExtensionSilentTokenRequest
 
+@synthesize requestCompletionBlock, operationRequest;
```

Comment:
Do we still need `@synthesize`? I remember it is auto-synthesize since quite some time ago, or maybe I missed something :)